bot/exts/halloween/spooknamerate.py

```diff
@@ -0,0 +1,338 @@
+# Importing everything
+import asyncio
+import calendar
+import json
+import logging
+import random
+from collections import defaultdict
+from datetime import datetime
+from pathlib import Path
+from typing import Dict
+
+from discord import Embed, Message, Reaction, TextChannel, User
+from discord.ext import tasks
+from discord.ext.commands import Bot, Cog, Context, group
+
+from bot.constants import Channels, Client, Colours, Month
+from bot.utils.decorators import InMonthCheckFailure
+
+logger = logging.getLogger(__name__)
+month_to_limit_to = Month.NOVEMBER
```

Comment:
Was this set for testing? Should be October afaik

Reply:
Oh no my bad